CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
 
+## master
+[v6.4.5...master](https://github.com/deployphp/deployer/compare/v6.4.5...master)
+
+### Changed
+- [composer 3] Removed deprecated option "--no-suggest"
```

Comment:
composer 2

Reply:
No. Composer 3:
![image](https://user-images.githubusercontent.com/10347617/96911019-f573cb80-14a8-11eb-84a8-6684829c9200.png)

Reply:
Hmm composer 2 is not even released.. Not sure it makes sense assumptions on what composer 3 will look like

Reply:
I will try to explain it so that Google Translate will understand.

I am currently using Composer 2.0.0 RC2. When the "composer update/install" command is executed with the `--no-suggest` switch, it warns that in "composer 2" this switch does nothing and is contained there only as backward compatibility.
He also warns that in composer 3 this key will be removed.
Composer 2 will not display package suggestions by default. To view them, you need to explicitly run the "composer suggests" command.

![image](https://user-images.githubusercontent.com/10347617/96913966-06264080-14ad-11eb-882e-508d19fcbd1b.png)